dart-runtime/lib/types.dart

```diff
@@ -207,6 +213,12 @@ decode(Map<String, Object> typeTable, String path, Object type, Object value) {
                 "Invalid Type at '$path', expected ${jsonEncode(type)}, got ${jsonEncode(value)}");
           }
           return DateTime.parse("${value}Z").toLocal();
+        case "void":
+          if (!(value != null)) {
+            throw SdkgenTypeException(
+                "Invalid Type at '$path', expected ${jsonEncode(type)}, got ${jsonEncode(value)}");
+          }
```

Comment:
Same as before:

This check is causing a breaking change with other clients, as it doesn't allow an API to change the return type of a method from void to any type. It should be removed :)